### PR TITLE
CHANGE: updated  readme file to include libraries needed to run cmake .

### PR DESCRIPTION
**Pull request by oyvang**
minor issue: readme file "## Dependencies ##
Please install these dependencies on your system." did not include boost, gtkmm and gtksourceview which is all needed to run cmake .

### CODE REVIEW

**Comment by oyvang** [comment]
Added changes, pluss some more. Please see: https://github.com/cppit/jucipp/commit/fd49094e4896fdfd18189191e1d386ca6a6394ca for changes

**Comment by zalox** [comment]
Please create docs/install.md in libclangmm project

* create <project-root-dir>/docs/install.md
* move install instructions to that file (from readme.md)
* link to this file in readme.md in both juci++ and libclanmm project

File: README.md [hunk 1]
```diff
@@ -26,15 +26,40 @@ Configuration description will arrive shortly after source code is added.
...
+* libpython-dev 
+* libclang-dev
 * [libclangmm](http://github.com/cppit/libclangmm/)
 * cmake
 * make
 * clang or gcc (compiler)
 
 ## Installation ##
+**Linux**
+```sh
```
Comment (zalox) [inline README.md]: This is not linux specific. This is debian spesific. Please change to debian.

**Comment by oyvang** [comment]
Is this something you where looking for? The link is broken at the moment, but will work when merging into master. The link include branch name, this is why it is not working on readme branch.

**Comment by oyvang** [comment]
I will wait with doing the same on libclangmm until this one is good.

**Comment by zalox** [comment]
Yes, just add link to the install guide in clangmm/docs/install.md then its good :dancer:

**Pull request merged**